src/Utilities/DeclarationBlockParser.php

```diff
@@ -43,12 +43,12 @@ public function normalizePropertyName(string $name): string
      *
      * The declaration block
      *
-     *   "color: #000; font-weight: bold;"
+     *   `color: #000; font-weight: bold;`
      *
      * will be parsed into the following array:
      *
-     *   "color" => "#000"
-     *   "font-weight" => "bold"
+     *   `"color" => "#000"`
+     *   `"font-weight" => "bold"`
```

Comment:
As we are changing this, I think we should make the example into an actual PHP array, and use single quotes for the strings:

```suggestion
     *   `['color' => '#000', 'font-weight' => 'bold']`
```

It might be tricky to put a multiline array in backticks in the DocBlock - it certainly is as a GitHub suggestion - but if you know a good way to do so, that would be better.